docs/css/custom.scss

```diff
@@ -0,0 +1,239 @@
+// out: custom.css
```

Comment:
If this .scss file just generates the .css file, I think we should not use it for now. There doesn't look like there is much difference between the files, so maintaining just the single .css file seems ok.